src/hooks/useClaimableWalletToken/useClaimableWalletToken.ts

```diff
@@ -14,20 +14,29 @@ const supplyControllerAddress = '0xc53af25f831f31ad6256a742b3f0905bc214a430'
 const supplyControllerInterface = new Interface(WALLETSupplyControllerABI)
 
 const useClaimableWalletToken = ({
+  fetch,
   accountId,
   network,
   addRequest,
   totalLifetimeRewards,
   walletUsdPrice
 }: UseClaimableWalletTokenProps): UseClaimableWalletTokenReturnType => {
+  const { isLoading, constants } = useFetchConstants({ fetch })
```

Comment:
I think it's better to pass either the `useFetchConstants` here, either the `constants` and `isLoading` params. Let's discuss this offline.